lighthouse-core/gather/driver.js

```diff
@@ -675,20 +697,26 @@ class Driver {
     });
   }
 
-  endNetworkCollect() {
+  endNetworkCollect(opts = {}) {
+    const pauseAfterLoadMs = (opts.flags && opts.flags.pauseAfterLoad) || PAUSE_AFTER_LOAD;
```

Comment:
don't love that we have to have this logic twice :/ also seems a little strange that we have to wait for idle again since we could have explicitly timed out trying to wait for network idle the first time, I'm guessing you want to be sure you're not giving up while it's being requested?